packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/ui_components/onboarding_error_listener.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:widget_toolkit/ui_components.dart';
+
+import '../../base/models/errors/error_model.dart';
+import '../blocs/onboarding_phone_bloc.dart';
+
+/// Widget that listens to the errors of the [OnboardingPhoneBloc]
+/// and shows a [showErrorBlurredBottomSheet] with the error message.
+class OnboardingErrorListener extends StatelessWidget {
+  const OnboardingErrorListener({super.key});
+
+  @override
+  Widget build(BuildContext context) =>
+      RxBlocListener<OnboardingPhoneBlocType, ErrorModel>(
+        state: (bloc) => bloc.states.errors,
+        listener: (context, error) => showErrorBlurredBottomSheet(
+          context: context,
+          error: _translateError(context, error),
+        ),
+      );
+
+  String _translateError(BuildContext context, Exception exception) {
```

Comment:
Please use the error built-in extensions, instead of using toString();. This will ensure the error is localized properly

Reply:
Done.